plugin/trino-hive-hadoop2/bin/run_hive_s3_tests.sh

```diff
@@ -10,9 +10,6 @@ check_vars S3_BUCKET S3_BUCKET_ENDPOINT \
     AWS_ACCESS_KEY_ID AWS_SECRET_ACCESS_KEY
 
 cleanup_hadoop_docker_containers
-
-# Use Hadoop version 3.1 for S3 tests as the JSON SerDe class is not available in lower versions.
-export HADOOP_BASE_IMAGE="ghcr.io/trinodb/testing/hdp3.1-hive"
```

Comment:
I assume this is not needed because we always use this version, right?

Reply:
Yes, exactly.